src/main/app/case/CosApiClient.ts

```diff
@@ -179,14 +179,16 @@ export class CosApiClient {
     user: UserDetails,
     caseId: string,
     partyId: string,
-    data: Partial<CaseData>
+    data: Partial<CaseData>,
+    isWelsh: boolean
   ): Promise<DocumentDetail> {
     try {
       const headers = {
         Accept: 'application/json',
         'Content-Type': 'application/json',
         Authorization: 'Bearer ' + user.accessToken,
         ServiceAuthorization: 'Bearer ' + getServiceAuthToken(),
+        isWelsh,
```

Comment:
Does this param need to go in headers? should it be ideally not in the request ?

Reply:
removed this from headers will send this flag as request parameter